### ISSUE TITLE
Fix undesired new lines after checking for Kitty/Sixel support

### ISSUE THREAD

**Issue by o2sh** [o2sh/onefetch#307]
both `kitty::KittyBackend::supported()` - if `KITTY_WINDOW_ID` is not set - and `sixel::SixelBackend::supported()` need to `println!()` a sequence of characters in order to check if the corresponding backend is supported, resulting in an empty line per check:

![out](https://user-images.githubusercontent.com/13710835/96910543-f0f6e500-149f-11eb-8724-123f255aa8cf.png)

![fail](https://user-images.githubusercontent.com/13710835/97933135-2f2eb700-1d72-11eb-8fd4-f0bd27b3c642.png)

or just one line if you use the `--image-backend` flag:

![fail2](https://user-images.githubusercontent.com/13710835/97933319-b4b26700-1d72-11eb-837f-f2b1c8dcaa4a.png)

Beyond esthetics, this seems to interfere with displaying images on xterm:

![xterm1](https://user-images.githubusercontent.com/13710835/97933823-1de6aa00-1d74-11eb-90bf-4da90847cc55.png)

with `--image-backend sixel` (only one additional new line):

![xterm2](https://user-images.githubusercontent.com/13710835/97933875-4bcbee80-1d74-11eb-98fb-bb16678dfaa2.png)

**Comment by yoichi** [comment]
In my environment, the behavior has not changed before and after f2bb5b0

**Comment by o2sh** [comment]
You're right @yoichi , with [`f2bb5b0`](https://github.com/o2sh/onefetch/commit/f2bb5b0f211061bc9059649b0bf2f8a9d8d1ff6b), I just removed the `ImageBackend::check_if_supported()` and added some error handling. Now, we don't have an added line when using `--image-backend`, but that's it.

However, I couldn't reproduce this part

> 
> Beyond esthetics, this seems to interfere with displaying images on xterm:
> 
> ![xterm1](https://user-images.githubusercontent.com/13710835/97933823-1de6aa00-1d74-11eb-90bf-4da90847cc55.png)

So, I closed the issue to further investigate instead of just updating it.
